Dockerfile

```diff
@@ -14,6 +14,31 @@ RUN apk add --no-cache protoc
 WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
+
+# Environment variables that must be set during build,
+# so that they are available in the bundles.
+#
+# I hate that Docker doesn't support *not* setting environment variables
+# when the argument isn't provided, so we now have duplicate defaults.
```

Comment:
Can we just set empty strings instead?

The other thing is that we don't necessarily need to pass all of these to the container process (some are only needed at build time)

Reply:
Sadly empty (`""`) environment variables still get parsed as set instead of `undefined` which makes handling these with defaults super annoying

Reply:
I've found a better solution.